PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingServiceEventHandlers.cs

```diff
@@ -149,7 +173,54 @@ private void Debugger_DebuggerStop(object sender, DebuggerStopEventArgs e)
             }
         }
 
-        private void ProcessDebuggingCommandResults(PSDataCollection<PSObject> output)
+        private PSDataCollection<PSObject> ExecuteDebuggingCommand()
+        {
+            PSCommand psCommand = new PSCommand();
+            psCommand.AddScript(_debuggingCommand);
+            psCommand.Commands[0].MergeMyResults(PipelineResultTypes.Error, PipelineResultTypes.Output);

```

Comment:
Shouldn't be as we did AddScript ahead, if it was not added in or added in but commands.count == 0 for some unknown reason, then I would just let the exception thrown.